src/utils.ts

```diff
@@ -142,3 +142,14 @@ export function convertNewlineToBr(text: string) {
 export function createMarkup(text: string) {
   return { __html: text };
 }
+
+export const getLocalStorage = (key: string) => {
+  const value = localStorage.getItem(key);
+  if (value !== null) {
+    return JSON.parse(value);
+  }
+};
+
+export const setLocalStorage = (key: string, value: string) => {
+  localStorage.setItem(key, JSON.stringify(value));
+};
```

Comment:
얘도 어차피 내부에서 `JSON.stringify`을 하기 때문에 `value: string`으로 받는 것이 의미가 없는 것 같아용!

```suggestion
export function setLocalStorage<T>(key: string, value: T) {
  localStorage.setItem(key, JSON.stringify(value));
};
```

요렇게 하면 다양한 타입의 `value`를 받을 수 있을 것 같슴다!

Reply:
아하 그렇네요..! 감사합니다 동욱님!